server/server.js

```diff
@@ -297,7 +301,7 @@ Server.prototype._getRateLimiterForMessageType = function(messageType) {
 
     if (!rateLimiter) {
       // TODO: subscribe, as rate limiter operation, should be configurable. 
-      rateLimiter = new RateLimiter(messageType.policy.limit);
+      rateLimiter = this.buildRateLimiter(messageType);
```

Comment:
I just removed it in favor of emitting an event every time radar server
adds / builds a new rate limiter.

Lucas Florio
Software Engineer
Zendesk <http://www.zendesk.com>

On Mon, Jun 22, 2015 at 5:17 PM, Vanchi Koduvayur <notifications@github.com>
wrote:

> In server/server.js
> <https://github.com/zendesk/radar/pull/142#discussion_r32998995>:
>
> > @@ -297,7 +301,7 @@ Server.prototype._getRateLimiterForMessageType = function(messageType) {
> >
> >      if (!rateLimiter) {
> >        // TODO: subscribe, as rate limiter operation, should be configurable.
> > -      rateLimiter = new RateLimiter(messageType.policy.limit);
> > +      rateLimiter = this.buildRateLimiter(messageType);
>
> I don't think this is actually needed. Let's not add more methods.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zendesk/radar/pull/142/files#r32998995>.
>